test/InterchainTokenService.js

```diff
@@ -2958,11 +2958,12 @@ describe('Interchain Token Service', () => {
         });
 
         it('Should be able to receive token only if it does not trigger the mint limit', async () => {
-            const tokenFlowLimit = await service.flowLimit(tokenId);
+            const tokenManager = await getContractAt('ITokenManager', await service.tokenManagerAddress(tokenId), wallet);
```

Comment:
```suggestion
            const tokenManager = await getContractAt('ITokenManager', await service.deployedTokenManager(tokenId), wallet);
```
only use `tokenManagerAddress` query for predictions for style